modules/nvidia-gpu-azure-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,530 @@
+          credentialsSecret:
+            name: azure-cloud-credentials
+            namespace: openshift-machine-api
+          diagnostics: {}
+          image:
+            offer: ""
+            publisher: ""
+            resourceID: /resourceGroups/myclustername-rg/providers/Microsoft.Compute/galleries/gallery_liqcui_azocp12_n6n4r/images/myclustername-gen2/versions/latest
+            sku: ""
+            version: ""
+          kind: AzureMachineProviderSpec
+          location: centralus
+          managedIdentity: myclustername-identity
+          metadata:
+            creationTimestamp: null
+          networkResourceGroup: myclustername-rg
+          osDisk:
+            diskSettings: {}
+            diskSizeGB: 128
+            managedDisk:
+              storageAccountType: Premium_LRS
+            osType: Linux
+          publicIP: false
+          publicLoadBalancer: myclustername
+          resourceGroup: myclustername-rg
+          spotVMOptions: {}
+          subnet: myclustername-worker-subnet
+          userDataSecret:
+            name: worker-user-data
+          vmSize:
+          vnet: myclustername-vnet
+          zone: "1"
+status:
+  availableReplicas: 1
+  fullyLabeledReplicas: 1
+  observedGeneration: 1
+  readyReplicas: 1
+  replicas: 1
+----
+
+. To verify your changes, perform a `diff` of the original compute definition and the new GPU-enabled node definition by running the following command:
++
+[source,terminal]
+----
+$ diff machineset-azure-ocp.4.12.2-020623.yaml machineset-azure-_GPU_ocp.4.12.2-020623.yaml
+----
++
+.Example output
+[source,terminal]
+----
+14c14
+<   name: myclustername-worker-centralus1
+---
+>   name: myclustername-nc4ast4-gpu-worker-centralus1
+23c23
+<       machine.openshift.io/cluster-api-machineset: myclustername-worker-centralus1
+---
+>       machine.openshift.io/cluster-api-machineset: myclustername-nc4ast4-gpu-worker-centralus1
+30c30
+<         machine.openshift.io/cluster-api-machineset: myclustername-worker-centralus1
+---
+>         machine.openshift.io/cluster-api-machineset: myclustername-nc4ast4-gpu-worker-centralus1
+67c67
+<           vmSize: Standard_D4s_v3
+---
+>           vmSize:
+----
+
+. Create the GPU-enabled compute machine set from the definition file by running the following command:
++
+[source,terminal]
+----
+oc create -f machineset-azure-_GPU_ocp.4.12.2-020623.yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+machineset.machine.openshift.io/myclustername-nc4ast4-gpu-worker-centralus1 created
+----
+
+. View the machines and machine sets that exist in the `openshift-machine-api` namespace
+by running the following command. Each compute machine set is associated with a
+different availability zone within the Azure region.
+The installer automatically load balances compute machines across availability zones.
++
+[source,terminal]
+----
+$ oc get machineset -n openshift-machine-api
+----
++
+.Example output
++
+[source,terminal]
+----
```

Comment:
Replace Line 300 to 318:
`
NAME                                             DESIRED   CURRENT   READY   AVAILABLE   AGE
clustername-n6n4r-nc4ast4-gpu-worker-centralus1   1         1         1       1           122m
clustername-n6n4r-worker-centralus1               1         1         1       1           8h
clustername-n6n4r-worker-centralus2               1         1         1       1           8h
clustername-n6n4r-worker-centralus3               1         1         1       1           8h
`

Reply:
Replaced.